docsite/docs/fal/fal-dbt/global-scripts.md

```diff
@@ -4,12 +4,11 @@ sidebar_position: 5
 
 # Running scripts not associated with a dbt model
 
-Typically a [fal script is associated with a dbt model](model-scripts.md), this is how the [context variable is populated](../../Reference/variables-and-functions.md#context-variable). However you may want to invoke scripts independent of a dbt model as well. This can be achieved by adding a script configuration similar to meta for models, but in the schema.yml top level:
+Typically a [fal script is associated with a dbt model](model-scripts.md), this is how the [context variable is populated](../reference/variables-and-functions.md#context-variable). However you may want to invoke scripts independent of a dbt model as well. This can be achieved by adding a script configuration similar to meta for models, but in the schema.yml top level:
 
 ```yaml
 models:
-...
-
+---
```

Comment:
This was a `...` meaning "there is more stuff here we don't care about". But `---` is for [separating documents](https://github.com/yaml/yaml-spec/blob/main/spec/1.2/markdown/spec.md#-structures)

Maybe make it

```suggestion
# your model declarations...

```

Reply:
I think this was the linter's doing :D